pkg/storage/segments/store.go

```diff
@@ -217,6 +217,20 @@ func (s *segmentStore) Get(ctx context.Context, path storj.Path) (
 			return nil, Meta{}, err
 		}
 
+		// calculate how many minimum nodes needed based on t = k + (n-o)k/o
+		o := pr.GetRemote().GetRedundancy()
```

Comment:
use `scheme` or `rs` (for redundancy scheme) as the variable name rather than `o`.

Reply:
modified as suggested